packages/story-editor/src/utils/usePreventWindowUnload.ts

```diff
@@ -57,7 +67,10 @@ function usePreventWindowUnload() {
   return setPreventUnload;
 }
 
+declare const WEB_STORIES_DISABLE_PREVENT: string;
 const shouldDisablePrevent =
   typeof WEB_STORIES_DISABLE_PREVENT !== 'undefined' &&
   WEB_STORIES_DISABLE_PREVENT === 'true';
-export default shouldDisablePrevent ? () => () => {} : usePreventWindowUnload;
+export default shouldDisablePrevent
+  ? () => () => undefined
+  : usePreventWindowUnload;
```

Comment:
Why is this needed?

Reply:
Are you asking why is the change `{}` to `undefined` needed?
Since `{}` is not allowed by the linting rules then using `undefined` instead. Originally I ignored the eslint error but Morten had the idea of using `undefined` instead. It doesn't really change anything.